### PR TITLE
perform vector simplication before comparisons

### PR DESCRIPTION
**Pull request by dqnykamp**

### CODE REVIEW

**Comment by dqnykamp** [comment]
PR built off https://github.com/Doenet/DoenetTools/pull/1959 and meant to be merged afterwards. Only the second commit needs to be reviewed as part of this PR.

File: src/Core/utils/checkEquality.js [hunk 1]
```diff
@@ -110,8 +116,18 @@ export default function checkEquality({
     if (symbolicEquality) {
       check_equality = function (a, b) {
 
-        let expr_a = normalize(a);
-        let expr_b = normalize(b);
+        // let expr_a = normalize(a);
```
Comment (jaltekruse) [inline src/Core/utils/checkEquality.js:119]: can delete this comment

Comment (dqnykamp) [reply src/Core/utils/checkEquality.js:119]: Done

**Review by jaltekruse — APPROVED** [review]
+1

**Pull request merged**